classes/author/DAO.inc.php

```diff
@@ -58,7 +61,7 @@ public function getAuthorsAlphabetizedByJournal($journalId = null, $initial = nu
 
         $supportedLocales = [];
         if ($journalId !== null) {
-            $journalDao = DAORegistry::getDAO('JournalDAO'); /* @var $journalDao JournalDAO */
+            $journalDao = DAORegistry::getDAO('JournalDAO'); /** @var $journalDao \JournalDAO */
```

Comment:
This is legacy until `JournalDAO` gets written in to the new toolset, but anyway, should be `\APP\journal\JournalDAO`

Reply:
Fixed